# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/whois-add_libintl.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- whois-4.6.13/Makefile.orig	2005-01-27 10:16:12.000000000 -0800
+++ whois-4.6.13/Makefile	2005-01-27 10:16:14.000000000 -0800
@@ -6,7 +6,7 @@
 #LDFLAGS=-lnsl -lsocket
 
 # obsolete linux (libc 5)
-#LDFLAGS=-lintl
+LDFLAGS=-lintl
 
 # FreeBSD
 #LDFLAGS=-L/usr/local/lib -lgnugetopt -lintl
